packages/components/src/mobile/bottom-sheet/ripple.native.js

```diff
@@ -0,0 +1,93 @@
+/**
+ * External dependencies
+ */
+import {
+	Platform,
+	TouchableOpacity,
+	TouchableNativeFeedback,
+	View,
+} from 'react-native';
+
+/**
+ * WordPress dependencies
+ */
+import { withPreferredColorScheme } from '@wordpress/compose';
+
+/**
+ * Internal dependencies
+ */
+import rippleStyles from './ripple.native.scss';
+
+const ANDROID_VERSION_LOLLIPOP = 21;
+const ANDROID_VERSION_PIE = 28;
+
+const TouchableRipple = ( {
+	style,
+	onPress,
+	disabled: disabledProp,
+	children,
+	activeOpacity,
+	getStylesFromColorScheme,
+	accessible,
+	accessibilityLabel,
+	accessibilityRole,
+	accessibilityHint,
+} ) => {
+	const borderless = false;
+
+	const isTouchableNativeSupported =
+		Platform.OS === 'android' &&
+		Platform.Version >= ANDROID_VERSION_LOLLIPOP;
+
+	const disabled = disabledProp || ! onPress;
+	const rippleColor = getStylesFromColorScheme(
+		rippleStyles.ripple,
+		rippleStyles.rippleDark
+	).backgroundColor;
+
+	if ( isTouchableNativeSupported ) {
+		// A workaround for ripple on Android P is to use useForeground + overflow: 'hidden'
+		// https://github.com/facebook/react-native/issues/6480
+		const useForeground =
```

Comment:
It will be always false since the `borderless` is hardcoded.

Reply:
fixed